<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>411</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APIthet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
    <widget class="QWidget" name="apiTab">
     <attribute name="title">
      <string>Add API</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditURL">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxHTTPMethods">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>10</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHeader">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>41</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHeaderVal">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEditPayload">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>90</y>
        <width>283</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Gothic</family>
       </font>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddHeader">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>40</y>
        <width>76</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Header</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRandParam">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>91</y>
        <width>105</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClear">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>45</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRandParam">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>111</y>
        <width>105</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>145</y>
        <width>55</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+ API List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRun">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>45</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>API List</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>311</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddHeader_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddHeader_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Load List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAddHeader_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>70</y>
        <width>76</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sequence</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowserResults">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>6</y>
        <width>398</width>
        <height>185</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QCheckBox" name="checkBoxProxy">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>9</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Proxy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditProxyHost">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>10</y>
        <width>287</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>120</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditUsername">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>33</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>44</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAuth">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>32</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Auth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>33</y>
        <width>170</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAbout">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>APIthet version 1.1.4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>An application that is used to assess security of APIs...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> Especially in order to harden the API backend</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>225</y>
      <width>398</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="1" margin="3"/>
 <resources/>
 <connections/>
</ui>
